src/ol/PluggableMap.js

```diff
@@ -1045,7 +1045,7 @@ class PluggableMap extends BaseObject {
       }
     }
     mapBrowserEvent.frameState = this.frameState_;
-    const interactionsArray = this.getInteractions().getArray();
+    const interactionsArray = this.getInteractions().getArray().slice();
     if (this.dispatchEvent(mapBrowserEvent) !== false) {
```

Comment:
Just checking is there a reason `interactionArray` isnt built in this `if` condition?